src/iptvsimple/data/Channel.cpp

```diff
@@ -125,6 +125,10 @@ void Channel::SetStreamURL(const std::string& url)
 
   if (StringUtils::StartsWith(url, HTTP_PREFIX) || StringUtils::StartsWith(url, HTTPS_PREFIX))
   {
+    if (!Settings::GetInstance().GetUserAgent().empty() && GetProperty("http-user-agent").empty())
+    {
```

Comment:
No need for brackets

Reply:
This style?

`if ()
  statement`

Reply:
Without brackets and 2 spaces tab right?

Reply:
Yes